### PR TITLE
fix #83261 ga.de

### LINKED ISSUE #83261: ga.de
### Issue URL (Anti Adblock Script)
 [https://ga.de/app/consent/#](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fga.de%2Fapp%2Fconsent%2F%23)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Firefox
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Adblock Warning Removal List<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Other:</b><br/> AdGuard DNS<br/><br/><b>Language-specific:</b><br/> AdGuard Russian, <br/>AdGuard German, <br/>RU AdList, <br/>RU AdList: Counters, <br/>RU AdList: BitBlock

### PR DESCRIPTION
**Pull request by wout-codes**
#83261

### CODE REVIEW

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -1830,6 +1830,7 @@ censor.net,fizzicseducation.com.au,fparf.ru,revouninstaller.com,logincasino.com,
 ! ###mc_embed_signup
 honeysanime.com,circuitdigest.com,worldhistory.org,bizreport.com,wristreview.com,kosicednes.sk,tinyprojects.dev,cambio16.com,voxukraine.org,pnas.org,cachevalleydaily.com,bisnis.com,ilpost.it,bsblog.info,alcoparty.com.ua,codastory.com,informazionefiscale.it,kerch.fm,cinemablend.com,prostofinovo.ru,comss.ru,majorgeeks.com,buhgalter.com.ua,providr.com,24tv.ua,babylonbee.com,dexerto.com,football365.com,menrec.com,netdoktor.de,ottawamatters.com,retail-insider.com,walterfootball.com,wweek.com,zozhnik.ru###mc_embed_signup
 !
+ga.de##.park-footer__newsletter
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt:1833]: We do not remove them from footers.

File: GermanFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -1,6 +1,9 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/83261
+ga.de#?#div.park-section:has( aside[data-portal-id^="mobilebanner_"])
```
Comment (Alex-302) [inline GermanFilter/sections/css_extended.txt:5]: Blocks too much. For example
image

**Review by Alex-302 — CHANGES_REQUESTED** [review]
Also missed push notification:
`ga.de##.park-floating-bell`

File: GermanFilter/sections/css_extended.txt (hunk 2 above)
Comment (wout-codes) [reply GermanFilter/sections/css_extended.txt:5]: @Alex-302 Does this work?
`ga.de#?#div.park-section:has( li.park-teaser-collection__list-item > aside[data-portal-id^="mobilebanner_"])`

Comment (Alex-302) [reply GermanFilter/sections/css_extended.txt:5]: It seems it works.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**